From 8229f4fb700ba4fcb2ec3e9956491bf5ee8c0ae2 Mon Sep 17 00:00:00 2001
From: Aaron Conole <aconole@redhat.com>
Date: Mon, 3 Aug 2020 15:17:31 -0400
Subject: [PATCH] lldptool: make extern

This should only exist per final linked object.

Signed-off-by: Aaron Conole <aconole@redhat.com>
[Retrieved from:
https://github.com/intel/openlldp/pull/55/commits/8229f4fb700ba4fcb2ec3e9956491bf5ee8c0ae2]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 include/lldptool.h | 2 +-
 lldptool.c         | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/lldptool.h b/include/lldptool.h
index c919873..a190009 100644
--- a/include/lldptool.h
+++ b/include/lldptool.h
@@ -29,7 +29,7 @@
 
 #include "clif.h"
 
-struct lldp_head lldp_cli_head;
+extern struct lldp_head lldp_cli_head;
 
 int clif_command(struct clif *clif, char *cmd, int raw);
 void print_raw_message(char *msg, int print);
diff --git a/lldptool.c b/lldptool.c
index 664a248..d76cc48 100644
--- a/lldptool.c
+++ b/lldptool.c
@@ -64,6 +64,7 @@
 #include "lldp_util.h"
 #include "lldpad_status.h"
 
+struct lldp_head lldp_cli_head;
 struct lldp_head lldp_mod_head;
 
 static int show_raw;
